projects/plugins/jetpack/modules/google-fonts/current/load-google-fonts.php

```diff
@@ -18,6 +18,22 @@
  * @return object[] The collection data of the Google Fonts.
  */
 function jetpack_get_google_fonts_data() {
+	/**
+	 * Filters the Google Fonts data before the default retrieval process.
+	 *
+	 * This filter allows short-circuiting the default Google Fonts data retrieval process.
+	 * Returning a non-null value from this filter will bypass the default retrieval
+	 * and return the filtered value instead.
+	 *
```

Comment:
I would recommend adding the module name to that docblock, so the filter is automatically added to the correct category in the Codex:

```suggestion
	 *
	 * @module google-fonts
	 *
```